CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contributing guidelines for the project
+
+First off, thank you for considering contributing to this project. It's people like you that make this project such a great tool. We appreciate your help!
+
+The following is a set of guidelines for contributing to this project. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Introduction
+
+This project is using TypeScript and [OCLIF](https://oclif.io/) to build the CLI. The project is using Jest for testing, ESLint for code formatting and linting.
```

Comment:
```suggestion
This project uses TypeScript and [OCLIF](https://oclif.io/) to build the CLI, Jest for testing, and ESLint for code formatting and linting.
```

Reply:
Modified in https://github.com/nimblehq/infrastructure-templates/pull/141/commits/1b6304955c0ca8410a0a48d67a6894b2b425383e